tests/backstop/config.js

```diff
@@ -23,7 +23,7 @@ const viewports = require( './viewports' );
 // This will be passed through with the `backstop` command run with docker.
 if ( process.argv.includes( '--docker' ) ) {
 	const hostname = require( './detect-storybook-host' );
-	process.argv.push( `--storybook-host=${ hostname }` );
+	process.argv.push( `--storybook-host=http://${ hostname }:9001/` );
```

Comment:
Why was this changed?

Reply:
It has been moved from the `scenarios.js` file to not break URLs when `./dist/` is used as a host.